src/main/java/gyro/google/compute/InstanceResource.java

```
@@ -438,6 +495,23 @@ public void copyFrom(Instance model) {
 
     }
 
+    private String getMetadataFingerprint() {
+        Compute client = createComputeClient();
+        String fingerprint = null;
+
+        try {
+            Instance instance = client.instances().get(getProjectId(), getZone(), getName()).execute();
+            if (instance.getMetadata() != null) {
+                fingerprint = instance.getMetadata().getFingerprint();
+            }
+
+        } catch (IOException ex) {
+            // ignore
+        }
```

Instead of catching and ignoring the exception, mark the method with throws